code/modules/religion/rites/musician_rites.dm

```diff
@@ -0,0 +1,102 @@
+/*********Base**********/
+/datum/religion_rites/song_tuner
+	name = "Tune Song"
+	desc = "this is a base."
+	ritual_length = 10 SECONDS
+	favor_cost = 10
+///if repeats count as continuations instead of a song's end, TRUE
+	var/repeats_okay = TRUE
+///visible message sent to indicate a song will have special properties
+	var/visible_message
+
+/datum/religion_rites/song_tuner/invoke_effect(mob/living/user, obj/structure/altar_of_gods/AOG)
+	user.AddComponent(/datum/component/smooth_tunes, src, repeats_okay)
+
+/**
+  * Perform the song effect.
+  *
+  * Arguments:
+  * * arg1 - Atom (parent) of the smooth_tunes component. This is limited to the compatible items of said component, which currently includes mobs and objects so we'll have to type appropriately.
+  */
+/datum/religion_rites/song_tuner/proc/song_effect(atom/A, datum/song/S)
+
+/*********Subtypes**********/
+
+/datum/religion_rites/song_tuner/evangelism
+	name = "Evangelical Hymn"
+	desc = "Spreads the word of your god, gaining favor for each non-holy listener."
+	favor_cost = 0
+
+/datum/religion_rites/song_tuner/evangelism/song_effect(atom/A, datum/song/S)
+	if(!S || !GLOB.religious_sect)
+		return
+	for(var/i in S.hearing_mobs)
+		if(i == A)
+			continue
+		if(!isliving(i))
+			continue //stinky ghosts
+		var/mob/living/L = i
+		if(L.mind.holy_role)
+			continue
+		GLOB.religious_sect.adjust_favor(0.2)
+
+/datum/religion_rites/song_tuner/sooth
+	name = "Soothing Hymn"
+	desc = "Sing a sweet song, healing bruises and burns around you."
+	repeats_okay = FALSE
+
+/datum/religion_rites/song_tuner/sooth/song_effect(atom/A, datum/song/S)
+	if(!S)
+		return
+	for(var/i in S.hearing_mobs)
+		if(i == A)
+			continue
+		if(!isliving(i))
+			continue
+		var/mob/living/L = i
+		var/healy_juice = 0.25
+		if(L.mind.holy_role)
+			healy_juice*=3
+		L.adjustBruteLoss(-healy_juice)
+		L.adjustFireLoss(-healy_juice)
+
+/datum/religion_rites/song_tuner/pain
+	name = "Sorrow Song"
+	desc = "Sing a melancholic song, hurting those around you. Works less effectively on fellow priests."
+	repeats_okay = FALSE
+
+/datum/religion_rites/song_tuner/pain/song_effect(atom/A, datum/song/S)
+	if(!S)
+		return
+	for(var/i in S.hearing_mobs)
+		if(i == A)
+			continue
+		if(!isliving(i))
+			continue
+		var/mob/living/L = i
+		var/pain_juice = 0.5
+		if(L.mind.holy_role)
```

Comment:
IMO, this should be checking antimagic, not whether or not you're a priest

maybe we can finally use that "holy" antimagic that bibles give for something other than defending against revenant magic?